pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -1,34 +1,51 @@
 :ruby
+  show_single_hero = Homepage.show_single_hero
+  show_single_hero_mobile_gap = Homepage.show_single_hero_mobile_gap
   heroes_arranged, hero_display_time = Homepage.get_heroes_arranged(request)
   actions = Homepage.get_actions
-  show_census_layout, show_census_announcement, census_announcement = Homepage.get_census(request)
```

Comment:
unrelated, but does this mean we can remove the census announcement code? I don't think we're referencing it elsewhere, but don't know if we will again.

Reply:
I did remove some of the related code in [homepage.rb](https://github.com/code-dot-org/code-dot-org/blob/0d543be0ce3690541c3bd2121770f3f88d011389/pegasus/src/homepage.rb) but mainly because we were only using half of the "census"-related functionality.  We were using the "census layout" (shorter hero, no rotation of the hero image) but not showing anything actually census-related, and so I thought I'd take this chance to clean that up now, figuring we could restore what was strictly necessary once we end this campaign.